client/src/components/competitor/StatsCharacter.tsx

```diff
@@ -43,9 +42,9 @@ function StatsCharacter() {
         </ul>
       </section>
       <Link
-        to="/"
+        to="/choose-god"
```

Comment:
`to="/choose-god"`

On doit mettre les routes en français

Reply:
Pas de verbe pour le nom de la route ?